### DISCUSSION TITLE
Would like to use Cake at work, and work wants to know what enterprises are using it?

### DISCUSSION THREAD

**Discussion by goblinfactory** [cake-build/cake#2950] [Q&A] [answered]
I would like to use Cake at work, to break our hard dependancy on Team City, so that we can test and run the full build pipeline on a developer machine, and work wants to know what enterprises are using it?

Apart from the obvious impressive nearly 8 Million downloads, is there a list of "who's using it" somewhere that I can show them?

**Comment by devlead** [comment, marked as answer]
Don't think we've got a curated list, but got some pretty recognizable users of Cake are i.e. Microsoft, Xamarin, GitHub, Octopus Deploy to name a few that got public repos using Cake. 

Searching for build.cake on GitHub will result a couple thousand repositories
https://github.com/search?l=&q=filename%3Abuild.cake&type=code
Probably find some well known entities there too. 

But that won't be the complete enterprise picture, as many of those repos are either private or hosted elsewhere.

I've personally introduced to some pretty well known enterprises, in finance, automotive and e-commerce sectors, but not in liberty to share those.

**Reply by goblinfactory** [reply]
that should do it :D
